### ISSUE TITLE
Unable to find route for Sepolia Testnet

### ISSUE THREAD

**Issue by dev3019** [wormhole-foundation/wormhole-sdk-ts#431]
Task - transfer custom token from Eth-sepolia to solana-devnet
Have tried building a solution from the examples in js.
Was able to reach upto the step where sourceToken is attested onto solana-devnet(able to see in explorer)
When trying to findRoutes() using resolver, it returns

> failed to find routes for AutomaticTokenBridge:  Error: Wormhole Token Bridge Relayer contract for domain Sepolia not found

> failed to find routes for AutomaticPortico:  Error: Unsupported chain, no contract addresses for: Sepolia

> For the transfer parameters, we found these routes:  []

```
async function getSigner(chain){
  let signer
  const platform = chain.platform.utils()._platform
  switch (platform) {
    case 'Solana':
      signer = await (await solana.default()).getSigner(await chain.getRpc(), 'private_key', {
        debug: true,
      })
      break;
    case 'Evm':
      signer = await (await evm.default()).getSigner(await chain.getRpc(), 'private_key')
      break;
  
    default:
      throw new Error("Unrecognized platform: " + platform);
      break;
  }
  return {
    chain,
    signer,
    address: Wormhole.chainAddress(chain.chain, signer.address())
  }
}
```

```
try {
    const wh = await wormhole('Testnet', [evm.default, solana.default])
    const sndChainCtx = wh.getChain('Sepolia')
    const rcvChainCtx = wh.getChain('Solana')
    
    const sender = await getSigner(sndChainCtx)
    const receiver = await getSigner(rcvChainCtx)

    const srcTB = await sndChainCtx.getTokenBridge()
    // console.log(sndChainCtx.config.rpc)

    const token = Wormhole.chainAddress('Sepolia','0x7E399fd2C4B260b1cBC66548c03753aBA9990665')
    // const txGenerator = srcTB.createAttestation(token.address)
    // const txIds = await signSendWait(sndChainCtx, txGenerator, sender.signer)
    const txIds = [
      {
        chain: 'Sepolia',
        txid: '0xe090a71807c7aa84f876a385860bdd20b6310ee9ed0ade81f50847bbed19207b'
      }
    ]

    const destTB = await rcvChainCtx.getTokenBridge()
    const wrapped = await destTB.getWrappedAsset(token)
    if(wrapped){
      console.log('Already Attested: ', wrapped)
    } else{
      console.log('Not Attested')
    }

    // create new resolver, passing the set of routes to consider
    const resolver = wh.resolver([
      routes.TokenBridgeRoute, // manual token bridge
      routes.AutomaticTokenBridgeRoute, // automatic token bridge
      routes.CCTPRoute, // manual CCTP
      routes.AutomaticCCTPRoute, // automatic CCTP
      routes.AutomaticPorticoRoute, // Native eth transfers
    ]);
    // what tokens are available on the source chain?
    const destTokens = await resolver.supportedDestinationTokens(token.address, sndChainCtx, rcvChainCtx);
    const destToken = destTokens[0]

    const tnsfrRoute = await routes.RouteTransferRequest.create(wh, {
      from: sender.address,
      to: receiver.address,
      source: token,
      destination: destToken
    })

    // resolve the transfer request to a set of routes that can perform it
    const foundRoutes = await resolver.findRoutes(tnsfrRoute);
    console.log("For the transfer parameters, we found these routes: ", foundRoutes);

    const bestRoute = foundRoutes[0]
    console.log("Selected: ", bestRoute)
    console.log(bestRoute.getDefaultOptions())
    // Specify the amount as a decimal string
    const amt = "0.001";
    // Create the transfer params for this request
    const transferParams = { amount: amt, options: { nativeGas: 0 } };

    // validate the transfer params passed, this returns a new type of ValidatedTransferParams
    // which (believe it or not) is a validated version of the input params
    // this new var must be passed to the next step, quote
    const validated = await bestRoute.validate(transferParams);
    if (!validated.valid) throw validated.error;
    console.log("Validated parameters: ", validated.params);

    // get a quote for the transfer, this too returns a new type that must
    // be passed to the next step, execute (if you like the quote)
    const quote = await bestRoute.quote(validated.params);
    if (!quote.success) throw quote.error;
    console.log("Best route quote: ", quote);

    // If you're sure you want to do this, set this to true
    const imSure = true;
    if (imSure) {
      // EXAMPLE_REQUEST_INITIATE
      // Now the transfer may be initiated
      // A receipt will be returned, guess what you gotta do with that?
      const receipt = await bestRoute.initiate(sender.signer, quote);
      console.log("Initiated transfer with receipt: ", receipt);
      // EXAMPLE_REQUEST_INITIATE

      // Kick off a wait log, if there is an opportunity to complete, this function will do it
      // see the implementation for how this works
      await routes.checkAndCompleteTransfer(bestRoute, receipt, receiver.signer);
    }

    return res.status(200).send('received')
    
  } catch (error) {
    console.error(error)
  }
```

**Comment by dev3019** [comment]
> The way this was intended to work was by overriding the tokenMap, though that is entirely undocumented.
> 
> While coming up with an example I found an issue addressed in #432

How can I then resolve my issue, as on sepolia evm both tokenBridge & relayer are deployed, but the libraries are unable to fetch it

**Comment by barnjamin** [comment]
try `0.5.2-beta.1`
